### PR TITLE
Azure service catalogs

### PR DESCRIPTION
**Pull request by sshveta**

### CODE REVIEW

**Comment by dajoRH** [comment]
Lint report for commit 4360ba2e7043595c10ff6fe7cfeb4401abe0101f:

Everything seems all right :smile: .
*CFME QE Bot*

File: cfme/tests/services/test_cloud_service_catalogs.py [hunk 1]
```diff
@@ -69,23 +69,41 @@ def test_cloud_catalog_item(setup_provider, provider, dialog, catalog, request,
...
-        vm_name=vm_name,
-        instance_type=provisioning['instance_type'],
-        availability_zone=provisioning['availability_zone'],
-        cloud_tenant=provisioning['cloud_tenant'],
-        cloud_network=provisioning['cloud_network'],
-        security_groups=[provisioning['security_group']],
-        provider_mgmt=provider.mgmt,
-        provider=provider.name,
-        guest_keypair=provisioning['guest_keypair'])
+    if provider.type == "azure":
```
Comment (psav) [inline cfme/tests/services/test_cloud_service_catalogs.py]:
```python
    data = dict(
        item_type=provisioning['item_type'],
        name=item_name,
        description="my catalog",
        display_in=True,
        catalog=catalog.name,
        dialog=dialog,
        catalog_name=image,
        vm_name=vm_name,
        provider_mgmt=provider.mgmt,
        provider=provider.name,
        instance_type=provisioning['instance_type'],
        security_groups=[provisioning['security_group']],
    )
    if provider.type == "azure":
        updates = dict(
            virtual_private_cloud=provisioning['virtual_private_cloud'],
            cloud_subnet=provisioning['cloud_subnet'],
            resource_group=[provisioning['resource_group']],
        )
    else:
        updates = dict(
            availability_zone=provisioning['availability_zone'],
            cloud_tenant=provisioning['cloud_tenant'],
            cloud_network=provisioning['cloud_network'],
            guest_keypair=provisioning['guest_keypair']
        )
    data.update(updates)
    cloud_catalog_item = cct.Instance(**data)
```

**Comment by dajoRH** [comment]
Lint report for commit 7fdbb5c8810659fdfc4575ab5d09650d4bdae466:

Everything seems all right :smile: .
*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit a12734e3490ca65390a6dd6d6c07dbd9f4fdefc0:

Everything seems all right :smile: .
*CFME QE Bot*

**Pull request merged**